schematic/manifest/generator.py

```diff
@@ -1655,8 +1639,27 @@ def create_manifests(
             use_annotations (bool, optional): whether to use annotations. Defaults to False.
 
```

Comment:
Change data_type -> data_types and dataset_id -> dataset_ids here

Reply:
Thanks. I have updated it.